politeiawww/api/cms/v1/v1.go

```diff
@@ -918,12 +918,14 @@ type AbridgedCMSUser struct {
 	Domain         DomainTypeT     `json:"domain"`
 	ContractorType ContractorTypeT `json:"contractortype"`
 	Username       string          `json:"username"`
+	BilledHours    []Hours         `json:"billedhours"`
 }
 
 // CMSUsers is used to request a list of CMS users given a filter.
 type CMSUsers struct {
 	Domain         DomainTypeT     `json:"domain"`
 	ContractorType ContractorTypeT `json:"contractortype"`
+	BillingHistory bool            `json:"billinghistory"` // Boolean whether to include previous billed hours for users in the same domain.
```

Comment:
Shouldn't the inclusion of this data be based on who is requesting the data?  The use of a boolean as a request param is typically a code smell that something is setup wrong.

Reply:
It's been a while for this PR.  I'll review what I'd done and aim to remove this bool.

Reply:
Ah OK so if I remember correctly I decided to add a flag to pull billable hours only when requested since I was adding the functionality to the CMSUsers request.  

I figured there were a lot of times when CMSUsers would be needed to be called but not necessarily need the billable hours (search results/populating dropdowns etc).  

I could remove the bool and always calculate the billable hours (depending on matching domain) for all matchedUsers.  Or I could just make a new request altogether.

Reply:
This seems to me like it should be its own route to give you more control over filtering params like date ranges and such.  Also, looking at the `Hours` struct, it seems too generic to gain any value from.  Isn't the purpose of being able to see other contractors billables to help prevent over billing?  How would that work if the hours aren't broken down by line item?

Reply:
Yah that's a reasonable assessment.  I've opened another PR that now allows all users to see invoices and invoice details that will give them an early view into these numbers.  I'll revisit this PR to see what we can do to make it more applicable for domain-user auditing.

Reply:
I think instead of hours, I may just go with labor line items since that seems like what we want.  From there we should be able to aggregate things as we choose

Reply:
Sounds right.  So what's the plan for this PR?

Reply:
I'll close this PR and reopen with the updated changes.